### ISSUE TITLE
What should i do with the data after training?

### ISSUE THREAD

**Issue by faris060** [starcolon/photo-auto-balance#18]
What should i do with the data after the training is done?

**Comment by tao-pr** [comment]
All subdirectories of `data/*` can be removed manually in case you want to. However, only `data/raw` is where the raw training data is kept before expanding into a larger training set.

**Comment by faris060** [comment]
Thank you for answering, So all the train data information is store in trainset.csv?Then the data is used to balance the image?

**Comment by tao-pr** [comment]
Exactly, all training vectors and their labelled values are kept as `trainset.csv`.

**Comment by faris060** [comment]
Thank you for answering.
